chapters/1.md

```diff
@@ -11,33 +11,39 @@ Maria is purchasing a car from George, and in this chapter we will consider thre
 
-Depending on how the funds were sent, it could take anywhere from minutes to days before the $2,500 shows up in George's bank account. Since George is not privy to the banks' ledgers or communications, the entire process is opaque and cannot be monitored.
+Depending on how the funds were sent, it could take anywhere from minutes to days before the $2,500 shows up in George's bank account. Since George is not privy to the banks' ledgers or internal communications, the entire process is opaque to him and cannot be monitored.
 
 Many people that have not personally experienced economic disruption take functioning banks and the validity of their IOUs for granted. Few individuals consider the unsettling ramifications of handing their lifelong savings to opaque corporations, often putting all their eggs in a single institutional basket. 
 
-Losses can occur due to many other issues. One clear example could be the neglicence of the bank operator (while, for example, shifting assets between two accounts). A bank operator had to face up work overloading, lacking of standards, and poor working conditions; even if they developed a system to automatically check for errors, some recent news reports prove the contrary. We have to mark that banks are based on partial inexistent fund and if all people widthdraw their money, the bank overextends their assets and goes out of business. Feelings of malice and corruption have also an impact on economical systems; a hacker thieves or a rogue employe could profit off their priveleges in order to drain all your assets.
+Losses can occur due to many other issues. Bank employees may simply make a mistake and monitoring software may fail to catch errors.  Bank employees, IT staff, etc might take advantage of their privileged position for personal gain, thus creating a much larger group which must be trusted when using a bank. Additionally, banks usually have debt obligations which far exceed the cash they hold and would collapse if even a small portion of their customers decided to withdraw their money at the same time (a so-called “bank run”).
 
-The availability of the bank have been always an issue for the customer: especially in the weekends and during national holidays (which varies from nation to nation), there are some practical difficulties due to closing of them. Many economical transactions sent in "bank holidays" could fail and you have to wait the opening to receive or send your funds. 
+Banks limit your access to your money. They may be closed on weekends and when there is large emergencies or during holidays (which vary from country to country).  If the bank's servers are shut down for maintenance or there is a bug in the e-banking system / app (not uncommon), customers have no recourse but to wait until it is made available again. This can lead to significant delays, frustration and loss of economic opportunities by those who rely on the banks.
```

Comment:
> bug

security issue

---
> e-banking system / app (not uncommon)

Suggested edit

> e-banking system or application....

I'd like to mention some examples of hacks occurred in history.

> As many recent cases report, these electronic systems are often hacked: a given example is [....] (Bank of India? Isle of Man? ).

Reply:
I've made the first two changes.

As for examples, the term "recent" becomes out of date quickly :-)
Although I fully expect new hacks to appear in the legacy system I am concerned that most of these are not well know by the general public.  They hear something about hacks on the news, but most "normal" people I know (people who don't work in this area) couldn't name a single recent hack of a bank, and even then, usually only something that directly affected them (and would not be known by people in another country often).

Example: Americans might know about Capital One or Equifax, but not Europeans.
This in india might know about the BOI, but not Americans.

Maybe we can make a statement about the regularity of data breaches (not just banks being hacked)?

Here are examples more people might relate to: https://informationisbeautiful.net/visualizations/worlds-biggest-data-breaches-hacks/

Facebook leaking 420 million people's records is something universal, but most others seem questionable for a worldwide audience unless we want to actually explain what happened.